easybuild/easyconfigs/b/boto3/boto3-1.35.36-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonBundle'
+
+name = 'boto3'
+version = '1.35.36'
+
+homepage = 'https://github.com/boto/boto3'
+description = """Boto3 is the Amazon Web Services (AWS) Software Development Kit
+(SDK) for Python, which allows Python developers to write software that makes
+use of services like Amazon S3 and Amazon EC2."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+builddependencies = [
+    ('binutils', '2.42'),
+]
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('Python-bundle-PyPI', '2024.06'),
+]
+
+exts_list = [
+    ('botocore', version, {
+        'checksums': ['354ec1b766f0029b5d6ff0c45d1a0f9e5007b7d2f3ec89bcdd755b208c5bc797'],
+    }),
+    ('jmespath', '1.0.1', {
+        'checksums': ['90261b206d6defd58fdd5e85f478bf633a2901798906be2ad389150c5c60edbe'],
+    }),
+    ('s3transfer', '0.10.0', {
+        'checksums': ['d0c8bbf672d5eebbe4e57945e23b972d963f07d82f661cabf678a5c88831595b'],
+    }),
+    (name, version, {
+        'checksums': ['586524b623e4fbbebe28b604c6205eb12f263cc4746bccb011562d07e217a4cb'],
+    }),
```

Comment:
I deliberately held back the version of boto3 because the latest version of aiobotocore is not compatible with botocore after 1.35.37 (yes, these packages are dependency hell par excellence).